### DISCUSSION TITLE
Broadlink2.0 integrieren

### DISCUSSION THREAD

**Discussion by psydie** [Zefau/ioBroker.jarvis#739] [Ideas / Feature Requests]
Hallo alle miteinander, ich habe JarVis jetzt seit kurzem und finde es super. 
Jetzt aber zu meinem Problem, Broadlink2 wird ja leider nicht unterstützt aber alle meine Thermostate laufen darüber.

Gibt es eine Möglichkeit diese dennoch mit einzubringen wenn auch über Umwege.

**Comment by Zefau** [comment]
Was genau funktioniert nicht?

Broadlink wird aktuell nicht über den Importer unterstützt, weshalb Geräte nicht automatisch erkannt werden.

Geräte können aber immer auch einfach manuell angelegt werden. Das sollte auch gehen.
